dressup/src/pages/ProductsPage/FilterSection/FilterSection.js

```diff
@@ -46,51 +57,101 @@ const FilterSection = () => {
         <ul className="category-list">
           <p>CATEGORIES</p>
           <li>
-            <input type="checkbox" name="categories" value="Tshirts"></input>
-            <label htmlFor="Tshirts"> Tshirts</label>
+            <input
+              type="checkbox"
+              name="Pants"
+              value="Pants"
+              checked={state.categories.Pants}
+              onChange={(e) => dispatch({ type: "catg", value: e.target.name })}
```

Comment:
Suggestion: use Descriptive naming, it's to have long name when they are making sense